src/main/java/core/levels/LevelManager.java

```diff
@@ -110,12 +118,50 @@ private void updateEnemies() {
         if (level.getCurrentTime() >= level.getSpawnTime()) {
             Spawner<Character> enemy = enemies.remove(0);
             enemy.spawn();
-            level.setEnemySpawns(enemies);
+            enemySpawns = enemies;
             level.setScore(level.getScore() + 1);
             level.setSpawnTime(level.getSpawnTime() + 15);
         }
     }
 
+    /**
+     * Create HashMap holding levelDifficulty as key, and number of enemies that correspond to
```

Comment:
levelDifficulty being a key makes no sense to me

why is it called levelDifficulty? then also levels with same difficulty would have the same key? also this is a refactor, adding LevelDifficulty seems to imply a whole new feature

just unclear what's it's used for